packages/backend/services/propertyService.js

```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
       });
     }
   });
+  propertiesWithinPolygons.forEach((property) => {
+    calculateTravelToWork(property);
+  });
   return [...new Set(propertiesWithinPolygons.flat())];
 };
 
-const getDistance = async (startCoordinates, endCoordinates, methodOfTravel) => {
+const getDirections = async (
+  startCoordinates,
+  endCoordinates,
+  methodOfTravel
+) => {
   const token = `?access_token=${process.env.MAPBOX_TOKEN}`;
-
   const { data } = await axios.get(
-    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates}${endCoordinates}?access_token=${token}`
+    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates};${endCoordinates}${token}`
```

Comment:
is the ; required between start and end coordinates?
```suggestion
    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates};${endCoordinates}?access_token=${token}`
```

Reply:
yes that's how they separate the start and end from each other. then the long and lat are separated with a comma. Go home it's late